apps/backoffice-v2/src/lib/blocks/hooks/useDocumentBlocks/useDocumentBlocks.tsx

```diff
@@ -358,6 +367,19 @@ export const useDocumentBlocks = ({
           })
           .cellAt(0, 0);
 
+        const documentEntries = Object.entries(
+          {
+            ...additionalProperties,
+            ...propertiesSchema?.properties,
+          } ?? {},
+        ).map(([title, formattedValue]) => {
+          if (isObject(formattedValue)) {
+            formattedValue.value ||= ocrResult?.parsedData?.[title];
```

Comment:
In React its more important not to make mutations. Just pass the value to the right instead of formattedValue.